naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/GraphQLController.scala

```diff
@@ -0,0 +1,92 @@
+package org.coursera.naptime.ari.graphql
```

Comment:
Small nit: do we need to include a license at the top of all of the files? (I think I've been missing that too)